.github/workflows/ci.yml

```diff
@@ -40,17 +48,17 @@ jobs:
         uses: browser-actions/setup-chrome@latest
 
       - name: Set up R version ${{ matrix.r-version }}
-        uses: r-lib/actions/setup-r@v2
+        uses: r-lib/actions/setup-r@v2.11.0
         with:
           r-version: ${{ matrix.r-version }}
 
       - name: Install dependencies (standard)
         if: matrix.python-version != '3.12.0-rc.2'
-        run: python -m pip install ".[test,hg]"
+        run: python -m pip install ".[test,hg,testR]"
```

Comment:
Why are these in square brackets?

Reply:
This is for the optional dependencies? i.e. essentially `pip install asv[test,hg,testR]`

Reply:
Thanks. Now I understand.